.github/workflows/nightly.yml

```diff
@@ -0,0 +1,42 @@
+name: Nightly tests
+
+on:
+    workflow_dispatch: # note: if started manually, it won't run all matrix
+    schedule:
+        - cron: "0 3 * * *"  #  Runs at 03:00 (3 AM) UTC every day
+
+concurrency:
+    group: nightly-${{ github.head_ref || github.ref }}-${{ inputs }}
```

Comment:
What is the meaning of inputs here?

Reply:
Oh, I need to restore input or remove it from there.
The goal was to distiguish the full and partial jobs.